### ISSUE TITLE
TypeError: cannot use 'in' operator to search for 'x'

### ISSUE THREAD

**Issue by jermlab** [taye/interact.js#731]
I was using a previous version of interact without issue, but when I switched to the current version, I'm getting load of errors: **TypeError: cannot use 'in' operator to search for 'x'**

I'm pretty sure the reason is that I'm hiding and removing interactable elements during an interaction so the movelistener is getting confused. I tried using unset() before removing elements, which worked, but I couldn't figure out how to re-set the interact.draggable for future elements of the same class. 

What would you recommend in this situation?

![errors](https://user-images.githubusercontent.com/51773223/59404510-ba0bbc00-8de1-11e9-803d-5239981b9df6.jpg)

Thanks.

**Comment by taye** [comment]
What restriction options do you set?

**Comment by jermlab** [comment]
That was the issue! Thanks.

**Comment by sydev** [comment]
Can you explain exactly what the issue was?

**Comment by jermlab** [comment]
I had a restriction class on a draggable, but the draggable parent <div> gets removed while in motion. Since the draggable had inertia= true, I think the draglistener was throwing the errors trying to check if the draggable was within the restriction class. If I moved the draggable slowly without "throwing" (high initial velocity), then there were no errors. The errors only appeared when quickly throwing the draggable and removing it while in motion. 

I was able to remove the restriction all together and work around it. 

This is truly an awesome and fun library, and I'm thankful it's out there. I'm looking forward to seeing the features in the paid version.
